src/test/app/Flow_test.cpp

```diff
@@ -1433,6 +1433,65 @@ struct Flow_test : public beast::unit_test::suite
             txflags(tfPartialPayment | tfNoRippleDirect));
     }
 
+    void
+    testRIPD1443(bool withFix)
+    {
+        testcase("ripd1443");
+
+        using namespace jtx;
+        Env env(*this, features(featureFlow));
+        {
+            auto closeTime = amendmentRIPD1298SoTime();
+            closeTime += env.closed()->info().closeTimeResolution;
+            env.close(closeTime);
+        }
```

Comment:
@wilsonianb A better way to write this would be to set the time to just before the fix for one test, then just after for the other.